tests/vmi_networking_test.go

```diff
@@ -762,11 +762,12 @@ var _ = Describe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 					ipv4NetworkCIDR = api.DefaultVMCIDR
 				}
 				By("Checking ping to gateway")
-				pingVirtualMachine(serverVMI, gatewayIPFromCIDR(ipv4NetworkCIDR), "\\$ ")
+				ipAddr := gatewayIPFromCIDR(ipv4NetworkCIDR)
+				Expect(tests.PingFromVMConsole(serverVMI, ipAddr, "\\$ ")).ToNot(HaveOccurred())
 
 				By("Checking ping to google")
-				pingVirtualMachine(serverVMI, "8.8.8.8", "\\$ ")
-				pingVirtualMachine(clientVMI, "google.com", "\\$ ")
+				Expect(tests.PingFromVMConsole(serverVMI, "8.8.8.8", "\\$ ")).ToNot(HaveOccurred())
```

Comment:
Nit: the notation "expect thing() to no have occurred" is quite cringeworthy, isn't there a better gomega way to express that we don't want any error?

Reply:
I agree. There is ‘Expect(thing()).To(Succeed())’

Reply:
Changed them all to the "to-succeed" expression.
Basically, if you assert directly an call, "to-succeed" fits well. If on the other hand you have an "error" variable and you assert it, then the "tonot-haveoccured" fits better.